girder_volview/utils.py

```diff
@@ -4,6 +4,8 @@
 
 SESSION_ZIP_EXTENSION = ".volview.zip"
 
+IGNORE_EXTENSIONS = [".yaml", ".svs"]
```

Comment:
We could probably add a much longer list here of image and video formats handled by Histomics or Dive, though it is challenging to enumerate them all.

Reply:
🕵️ Dug up a few more:
```
VIDEO_EXTENSIONS = [
    ".mp4",
    ".webm",
    ".avi",
    ".mov",
    ".wmv",
    ".mpg",
    ".mpeg",
    ".mp2",
    ".ogg",
    ".flv",
]
LARGE_IMAGE_EXTENSIONS = [".svs", ".vms", ".vmu", ".ndpi", ".scn"]  # ".tif", ".tiff",
```